packages/validator/test/cli-validator/mock-files/oas3/clean-with-tabs.yml

```diff
@@ -31,13 +31,15 @@ paths:
 						format: int32
 						default: 10
 						maximum: 100
-				- name: offset
+				- name: start
```

Comment:
I guess we chose the wrong style of pagination for our "clean" API definition :joy: